added release notes up to 10.15.327.0 (#83)
SoftwareAG/c8y-releasenotes: content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

<td>MTM-47879</td>
</tr>

<tr>
<td>Administration</td>

> <td>Files of shared applications are now accessible to all subtenants of the owner tenant. This supports a "self-service" approach for web applications (subtenants can decide if they want to install a shared application) as opposed to the currently used concept (only a parent tenant can decide if a web application gets subscribed to a subtenant).

Reviewer: ```suggestion
<td>Files of shared applications are now accessible to all subtenants of the owner tenant. This supports a "self-service" approach for web applications (subtenants can decide if they want to install a shared application) as opposed to the current approach (only a parent tenant can decide if a web application gets subscribed to a subtenant).
```

the information in parentheses could be added as "where" relative clauses instead.

<br>
Technical details:
<br>Static files from web applications with SHARED availability are now resolved and served similarly to those with MARKET subscribed. The feature covers both paths: <code>&lt;url&gt;/apps/&lt;app-name&gt;/&lt;file-path&gt;</code> and <code>&lt;url&gt;/apps/public/&lt;app-name&gt;/&lt;file-path&gt;</code>. MARKET applications have a higher priority than SHARED applications if the context path is duplicated in tenant and parent tenant(s), but SHARED applications are served regardless of the subscription.</td>
<td>MTM-48150</td>
</tr>

<td>
Authentication</td>
<td>In the <b>Authentication</b> page, if the forbidden or trusted user agents lists for a specific authentication configuration are empty, the removal button will no longer be displayed. Related inconsistencies in previous versions have been fixed. </td>
<td>
MTM-43723</td>
</tr>

<td>
Authentication</td>

> <td>Trial tenants created in Software AG Cloud are by default not allowed to access the single-sign on configuration in Cumulocity IoT to prevent its potential misconfiguration by users. A REST endpoint is used by the Management tenant to restrict or allow the access to the configuration for specific tenants. Refer to <a href="https://cumulocity.com/guides/10.16.0/users-guide/administration/#configuration-access">Administration > Single sign-on > Configuration access</a> in the <i>User guide</i> for more details. </td>

Reviewer: ```suggestion
<td>Trial tenants created in Software AG Cloud are by default not allowed to access the single sign-on configuration in Cumulocity IoT to prevent its potential misconfiguration by users. A REST endpoint is used by the Management tenant to restrict or allow the access to the configuration for specific tenants. Refer to <a href="https://cumulocity.com/guides/10.16.0/users-guide/administration/#configuration-access">Administration > Single sign-on > Configuration access</a> in the <i>User guide</i> for more details. </td>

```

<tr>
<td>
REST API</td>

> <td>The <code>detail()</code> and <code>delete()</code> method of the @c8y/client MeasurementService is shown as deprecated as it is not supported with the new timeseries.</td>

Reviewer: is it only shown as deprecated or is it really deprecated? (curious)

<td>MTM-48008</td>
</tr>

<tr>
<td>Data broker</td>

> <td>The data broker connection handling between source and target tenants has been improved. Excessive resource usage for connectors using the operations API was reduced and it is now properly cleaned up.</td>

Reviewer: ```suggestion
<td>The data broker connection handling between source and target tenants has been improved. Excessive resource usage for connectors using the operations API has been reduced and it is now properly cleaned up.</td>

```
